main.js

```diff
@@ -693,33 +690,35 @@ class Sourceanalytix extends utils.Adapter {
 
 			default:
 				this.log.error("Error in case handling of cost type identificaton : " + obj_cust.state_type);
-				skip_calc = true;
+				return;
```

Comment:
Returning instead of the if branch allows TypeScript to know that `cost_unit` is defined.